ui/qt-app/Glue.hs

```diff
@@ -136,6 +136,9 @@ knitFaceToUI UiFace{..} KnitFace{..} =
       UiNewAddress -> do
         Right $ Knit.ExprProcCall
           (Knit.ProcCall Knit.newAddressCommandName [])
+      UiRemoveCurrentItem -> do
```

Comment:
Should this command also have a corresponding result? I don't think so, if only to show error.

Reply:
> I don't think so

same